Week3/homework/step2-7.js

```diff
@@ -21,3 +21,9 @@ f2(y);
 console.log(y);
 
 // Add your explanation as a comment here
+// In the first example, x is a global variable meaning that it can be accessed in and out of the function. When the
+// value of x is passed in the function, the function adds one to it and returns a value of 10.
+
+// In the second function, the value of y is an object containing a key and a value.
+// when the object is passed in the function, one is added to the value of the object itself
+//  and the value of the value increases by 1 and we get  {x: 10}
```

Comment:
I think you get it (?). Another example: 
```
let a = 4
let b = a
a += 1
What's the value of b?
```
vs objects...
```
let a = { name: 'gabe' }
b = a
a.country = 'us'
what's the value of b?
```
Do you see the difference?